pokeapp/src/components/badges/bagdeIcon.js

```diff
@@ -0,0 +1,201 @@
+// import VolcanoBadge from "../../assets/badges/Volcanobadge.png";
+// import VoltageBadge from "../../assets/badges/Voltage_Badge.png";
+// import WaterBadge from "../../assets/badges/WaterBadge.png";
+// import ZephyrBadge from "../../assets/badges/Zephyrbadge.png";
+
+/**
+ * Badge Model:
+ * name: string,
+ * icon: string,
+ * progressTarget: number,
+ * currentProgress: number,
+ * objectives: object,
+ */
+export const allBadges = [
+  {
+    name: "BasicBadge",
+    icon: BasicBadge,
+    progressTarget: 1,
+    currentProgress: 0,
+    objectives: {
+      bulblsaurOwned: false,
+      squirtleOwned: false,
+      charmanderOwned: false,
+    },
+  },
+  {
+    name: "GlacierBadge",
+    icon: GlacierBadge,
+    progressTarget: 1,
+    currentProgress: 0,
+    objectives: {
+      laprasOwned: false,
+      mamoswineOwned: false,
+      jynxOwned: false,
+      dewgongOwned: false,
+      cloysterOwned: false,
+      weavileOwned: false,
+    },
+  },
+  {
+    name: "RainbowBadge",
+    icon: RainbowBadge,
+    progressTarget: 1,
+    currentProgress: 0,
+    objectives: {
+      typeFireOwned: false,
+      typeWaterOwned: false,
+      typeIceOwned: false,
+      typeGrassOwned: false,
+      typeDarkOwned: false,
+      typeFairyOwned: false,
+      typeNormalOwned: false,
+      typeGroundOwned: false,
+      typeSteelOwned: false,
+      typeGhostOwned: false,
+      typePoisonOwned: false,
+      typeRockOwned: false,
+      typeBugOwned: false,
+      typePsychicOwned: false,
+      typeElectricOwned: false,
+      typeFlyingOwned: false,
+      typeDragonOwned: false,
+      typeFightingOwned: false,
+    },
+  },
+  {
+    name: "HeatBadge",
+    icon: HeatBadge,
+    progressTarget: 10,
+    currentProgress: 0,
+    objectives: {
+      tenFirePokemonOwned: false,
+      twentyFirePokemonOwned: false,
+      thirtyFirePokemonOwned: false,
+      fortyFirePokemonOwned: false,
+      fiftyFirePokemonOwned: false,
+      sixtyFirePokemonOwned: false,
+      seventyFirePokemonOwned: false,
+      eightyFirePokemonOwned: false,
+      ninetyFirePokemonOwned: false,
+      hundrenFirePokemonOwned: false,
+    },
+  },
+  {
+    name: "BoulderBadge",
+    icon: BoulderBadge,
+    progressTarget: 1,
+    currentProgress: 0,
+    objectives: {
+      kabutopsOwned: false,
+      onixOwned: false,
+      omastarOwned: false,
+      rhyhornOwned: false,
+      gravelerOwned: false,
+    },
+  },
+  {
+    name: "KnuckleBadge",
+    icon: KnuckleBadge,
+    progressTarget: 10,
+    currentProgress: 0,
+    objectives: {
+      tenFightingPokemonOwned: false,
+      twentyFightingPokemonOwned: false,
+      thirtyFightingPokemonOwned: false,
+      fortyFightingPokemonOwned: false,
+      fiftyFightingPokemonOwned: false,
+      sixtyFightingPokemonOwned: false,
+      seventyFightingPokemonOwned: false,
+      eightyFightingPokemonOwned: false,
+      ninetyFightingPokemonOwned: false,
+      hundrenFightingPokemonOwned: false,
```

Comment:
Same typo here.

Reply:
Consistency is key. Thanks again for the find.